bin/dump_data.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# Dump component data
+# This script is not intended for a data migration to another Objects Api instance.
+# Run this script from the root of the repository
+#
+# Note that postgres 17 requires postgres-client-17
```

Comment:
I forgot, but is it possible to dump a postgres <17 database with postgres-client-17? Or is it not backwards compatible?